### ISSUE TITLE
Plugin crashes upon start (if logging is disabled)

### ISSUE THREAD

**Issue by LeLunZ** [smhex/homebridge-wled-ws#75]
**Describe The Bug:**
Crashes upon start. The interesting part: for me it only happens if logging is disabled. 
Haven't really investigated it further as its quite late. But I can take another look at some other time.

**To Reproduce:**
I just installed the plugin, entered my ip and the name (didnt enter anything for the presets). (I am using the raspberry pi 5 with the newest firmware)

**Expected behavior:**
Shouldn't crash.

**Logs:**

```
[5/12/2024, 2:29:30 AM] TypeError: Converting circular structure to JSON
    --> starting at object with constructor 'Socket'
    |     property 'parser' -> object with constructor 'HTTPParser'
    --- property 'socket' closes the circle
    at JSON.stringify (<anonymous>)
    at WledWsPlatformAccessory.updateAccessoryInformation (/var/lib/homebridge/node_modules/homebridge-wled-ws/src/WledWsAccessory.ts:583:61)
    at WledWsPlatformAccessory.onStateReceived (/var/lib/homebridge/node_modules/homebridge-wled-ws/src/WledWsAccessory.ts:298:12)
    at /var/lib/homebridge/node_modules/homebridge-wled-ws/src/WledWsAccessory.ts:244:12
    at _class2.<anonymous> (/var/lib/homebridge/node_modules/homebridge-wled-ws/node_modules/wled-client/src/utils.emitter.ts:17:37)
    at _class2.[nodejs.internal.kHybridDispatch] (node:internal/event_target:826:20)
    at _class2.dispatchEvent (node:internal/event_target:761:26)
    at _class2.emit (/var/lib/homebridge/node_modules/homebridge-wled-ws/node_modules/wled-client/src/utils.emitter.ts:34:15)
    at _class2.setContext (/var/lib/homebridge/node_modules/homebridge-wled-ws/node_modules/wled-client/src/client.ts:186:19)
    at _class2.refreshContext (/var/lib/homebridge/node_modules/homebridge-wled-ws/node_modules/wled-client/src/client.ts:125:8)
```

**Plugin Config:**

```json
{
    "name": "wled-ws platform",
    "controllers": [
        {
            "name": "TV Licht",
            "address": "192.168.0.37"
        }
    ],
    "logging": false,
    "platform": "wled-ws"
}
```

**Environment:**

* **Plugin Version**: v0.0.11
* **Homebridge Version**: Homebridge v1.8.1 (HAP v0.12.0)
* **Node.js Version**: v20.12.2
* **NPM Version**: 10.5.0
* **Operating System**: Raspbian

**Comment by smhex** [comment]
Hi @LeLunZ,

I will try to reproduce it in my environment. I am using different versions of Node/Homebridge. Just to rule out the WLED side - which firmware does your WLED controller run on?

**Comment by LeLunZ** [comment]
I am running 0.14.3 :)

**Comment by smhex** [comment]
I did a first test using the same software versions like in your setup. The only difference is that my home bridge instance runs as docker container on a Synology NAS. I was not able to reproduce the crash. Do you have DEBUG Mode enabled in your HB instance?

**Comment by LeLunZ** [comment]
@smhex thx for trying. I think I have debug mode disabled(?), as I didn't change anything and I did a complete new install of the raspberry/homebridge and wled-ws.

But I will try to debug the issue and actually check what object goes into the JSON.stringify function. From the error, it seems its somehow referencing itself.

**Comment by LeLunZ** [comment]
@smhex enabling logging doesn't solve the issue but its just happening way less often.
Also it seems to work sometimes, when logging is disabled. On my last try, It crashed and rebooted homebridge 8 times, and then it worked. So it seems like some kind of race condition?

**Comment by smhex** [comment]
@LeLunZ it looks like one 🥹. I have an idea what might be the problem. I will create a beta for you to test my assumption.

**Comment by LeLunZ** [comment]
@smhex That works! No more crashes :)

**Comment by smhex** [comment]
Perfect, I think it's a good time to make a release out of this beta version 😄

**Comment by smhex** [comment]
Fixed in 0.0.13
